app/src/main/java/org/mozilla/tv/firefox/telemetry/TelemetryIntegration.kt

```diff
@@ -164,17 +164,17 @@ open class TelemetryIntegration protected constructor(
             return
         }
 
-        (TelemetryHolder.get()
-                .getPingBuilder(TelemetryPocketEventPingBuilder.TYPE) as TelemetryPocketEventPingBuilder)
-                .eventsMeasurement.add(this)
+//        (TelemetryHolder.get()
+//                .getPingBuilder(TelemetryPocketEventPingBuilder.TYPE) as TelemetryPocketEventPingBuilder)
+//                .eventsMeasurement.add(this)
     }
 
     private fun queueSessionMeasurements(context: Context) {
         TelemetryHomeTileUniqueClickPerSessionCounter.queueEvent(context)
         TelemetryEvent.create(Category.AGGREGATE, Method.CLICK, Object.POCKET_VIDEO,
                 pocketUniqueClickedVideoIDs.size.toString()).queue()
-        queuePocketVideoImpressionEvent()
-        queuePocketVideoClickEvent()
```

Comment:
I guess we'll lose the Pocket video click event, but that should be fine if it's just for a single release.

Reply:
It will be available `clickId.size()` will still send through `mobile-events` as it has always been